assets/js/index.js

```diff
@@ -6,74 +6,86 @@ const addBtn = document.getElementById('add-btn');
 
 let booksArr = [];
 
-// Remove a Book
-function removeBook() {
-  booksArr = booksArr.filter((book) => +book.id !== +this.parentElement.id);
+// Prevent Form from Submit
+bookForm.addEventListener('submit', (event) => {
+  event.preventDefault();
+});
 
-  this.parentElement.remove();
+class Book {
+  constructor(id, title, author) {
+    this.id = id;
+    this.title = title;
+    this.author = author;
+  }
 
-  booksArr.forEach((book, i) => {
-    book.id = i;
-    bookList.children[i].id = i;
-  });
+  // Remove a Book
+  static remove() {
+    booksArr = booksArr.filter((book) => +book.id !== +this.parentElement.id);
+    this.parentElement.remove();
 
-  localStorage.setItem('Books', JSON.stringify(booksArr));
-}
+    booksArr.forEach((book, i) => {
+      book.id = i;
+      bookList.children[i].id = i;
+    });
```

Comment:
- [ ] Books array should inside the class. Please change it. [Requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m2_plain_js_classes_v1_1.md#:~:text=You%20can%20use%20objects%20and%20arrays%2C%20but%20do%20it%20inside%20classes.)